boards/arm/nrf21540dk_nrf52840/nrf21540dk_nrf52840.dts

```diff
@@ -95,8 +95,8 @@
 			   <17 0 &gpio1 13 0>,	/* D11 */
 			   <18 0 &gpio1 14 0>,	/* D12 */
 			   <19 0 &gpio1 15 0>,	/* D13 */
-			   <20 0 &gpio0 26 0>,	/* D14 */
-			   <21 0 &gpio0 27 0>;	/* D15 */
+			   <20 0 &gpio0 0 0>,	/* D14 */
+			   <21 0 &gpio0 1 0>;	/* D15 */
```

Comment:
This does not look like a correct change. Pins P0.00/P0.01 are LFXO pins (XL1/XL2). And those two here are supposed to  be SDA/SCL lines in the Arduino connector.

Reply:
In the documentation that I have, those pins are mapped to both XL1/XL2 and D14/D15. Could it be an oversight in the schematic? I'm looking at revision 0.9.0 with date 09.07.2020

Reply:
I've verified on revision 0.7.0 and it seems it should be `10` and `9` accordingly. Not sure about newer DKs, because I don't have any.

Reply:
> In the documentation that I have, those pins are mapped to both XL1/XL2 and D14/D15. Could it be an oversight in the schematic? I'm looking at revision 0.9.0 with date 09.07.2020

I'm looking at the same one. Lines D14/D15 there (in Auxiliary Connector P12) are a different thing than D14/D15 here (these two correspond to SDA/SCL lines in Arduino Interface Connector P10 in the schematic).

Reply:
It seems there was some discrepancies in pin definitions, all cleared up now.